src/test/scala/edu/berkeley/ce/rockslicing/BlockSpec.scala

```diff
@@ -637,30 +643,32 @@ class BlockSpec extends FunSuite {
 
   test("Volume of two cube with extra plane should should be 4.0") {
     val redundantBoundingFaces = boundingFaces3 ++ List(
-      Face((-1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0)), 0.0, phi=0, cohesion=0)
+      Face(Array(-1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0)), 0.0, phi=0, cohesion=0)
     )
-    val redundantTwoCube = Block((0.0, 0.0, 0.0), redundantBoundingFaces)
-    val nonRedundantTwoCube = Block((0.0, 0.0, 0.0), redundantTwoCube.nonRedundantFaces)
+    val redundantTwoCube = Block(Array(0.0, 0.0, 0.0), redundantBoundingFaces)
+    // TODO this causes the test to fail, but shouldn't it?
```

Comment:
Looking at this test and the two that follow, it looks like they are actually poorly posed. By adding the additional face to `boundingFaces3`, it actually chops the block up into two blocks. When calling the `volume` method from `Block` it finds the volume for one of these blocks. The goal of these tests was to make sure that it's finding the correct volume for the `unitCube` and `twoCube` when they're cut in half, but I don't think these tests are actually testing that. 
So, perhaps we should just remove them or change it by adding the additional face by calling `cut` on the `unitCube` and `twoCube` before calling `volume` on the child blocks.